docs/ecosystem/account-abstraction/pimlico.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Pimlico
+sidebar_position: 1
+---
+
+Pimlico is a full-fledged account abstraction infrastructure platform that provides a set of advanced tools to help you build with smart accounts.
+
+Account abstraction is a feature that allows users to customize their Ethereum accounts using smart contract logic. This unlocks new capabilities, such as multi-sig, social recovery, meta-transactions, batched transactions, and paying for transactions with ERC-20 tokens.
+
+Pimlico offers a wide range of tools to help you get started on your smart account journey including:
+- Bundler
+- Paymaster
+- Comprehensive metrics and analytics
+- Advancement debugging / userOperation simulation
+- Sponsorship policies
+- Permissionless.js
+
+## Integrating Pimlico
+
+Pimlico and Account Abstraction can be easily added to existing projects or used to build new ones.
+
+To get started, check out our comprehensive [documentation](https://docs.pimlico.io). You can find things such as:
+
+- [Tutorials](https://docs.pimlico.io/permissionless/tutorial/tutorial-1) on how to integrate paymasters to make gasless transactions.
+
+- [Docs](https://docs.pimlico.io/permissionless) on how to use [Permissionless.js](https://docs.pimlico.io/permissionless), our vendor neutral SDK that works with all popular smart account types.
+
+- [Tips](https://docs.pimlico.io/infra/bundler/entrypoint-errors/aa33) on how to debug and solve common errors
+
+- [Guides](https://docs.pimlico.io/infra/platform/sponsorship-policies) on how to use setup custom sponsorship policies to finetune exactly what type of transactions your application wants to sponsor.
+
+- [Pimlico Dashboard](https://dashboard.pimlico.io) to view all your analytics and metrics as well as debugging tools.
+
+## Developers and community
+If you have any questions or issues, contact us on the following platforms:
+
+- [Website](https://www.pimlico.io/)
+- [Telegram](t.me/pimlicohq)
```

Comment:
Hi, can we change the telegram link to `https://t.me/pimlicohq`. Please note `https://` prefix. Otherwise the build fails